INSTALLING.md

```diff
@@ -2,199 +2,262 @@
-./build_metal.sh
+git clone https://github.com/tenstorrent/tt-metal.git --recurse-submodules
+cd tt-metal
+git submodule foreach 'git lfs fetch --all && git lfs pull'
+```
 
-# If you would like an out-of-the-box virtual environment to use,
+### Step 2. Setup Environment Variables and Invoke our Build Scripts:
+
+- Note: Some advanced build configurations like unity builds require `CMake 3.20`.
+  To install `CMake 3.20` run:
+    ```sh
+    sudo apt remove cmake -y
+    pip3 install cmake --upgrade
+    hash -r
+    cmake --version
+    ```
+
+- Run the appropriate command for the Tenstorrent card you have installed:
+
+| Card             | Command                              |
+|------------------|--------------------------------------|
+| Grayskull        | ```export ARCH_NAME=grayskull```     |
+| Wormhole         | ```export ARCH_NAME=wormhole_b0```   |
+| Blackhole        | ```export ARCH_NAME=blackhole```     |
+
+- Then run: 
+  ```
+  export TT_METAL_HOME=$(pwd)
+  export PYTHONPATH=$(pwd)
+  ./build_metal.sh
+  ```
+
+- (recomended) For an out-of-the-box virtual environment to use, execute:
+```
 ./create_venv.sh
 source python_env/bin/activate
 ```
 
-where `ARCH_NAME` is one of `grayskull`, `wormhole_b0`, or `blackhole`,
-depending on your Tenstorrent card type.
+---
 
-> [!NOTE]
->
-> Note about Python environments: You do not have to use `create_venv.sh`. If you
-> are less familiar with Python and its various environment tools, just use
-> `create_venv.sh` as shown above and the pre-built environment. If you choose
-> to install in your custom environment, please note that you may run into
-> compatibility issues between dependencies. It is up to the user to ensure
-> that all the packages in their environment are compatible with each other.
->
-> If you do choose to manage your own environment, please note that you must
-> use Pip 20.1.1 or lower to install this project. This is the highest version
-> of Pip that supports editable installs in the way that we use it.
+## Option 2: From Docker Release Image
+Installing from Docker Release Image is the quickest way to access our APIs and to start runnig AI models.
 
-#### Option 2: From wheel
+### Step 1. Download and Install the Latest Docker release Image:
 
-Download the latest wheel from our
-[releases](https://github.com/tenstorrent/tt-metal/releases/latest) page for
-the particular Tenstorrent card architecture that you have installed on your
-system. (ie. Grayskull, Wormhole, etc)
+- Run the command for the Tenstorrent card architecture you have installed:
 
-Install the wheel using your Python environment manager of choice. For example,
-to install with `pip`:
+  - For Grayskull:
+  ```
+  docker pull ghcr.io/tenstorrent/tt-metal/tt-metalium-ubuntu-20.04-amd64-release/grayskull:latest-rc
+  docker run --it --rm -v /dev/hugepages-1G:/dev/hugepages-1G --device /dev/tenstorrent ghcr.io/tenstorrent/tt-metal/tt-metalium-ubuntu-20.04-amd64-release/grayskull:latest-rc bash
```

Comment:
For all of the three cards, there is a mistake (by me) where I use `--it` instead of `-it`. One dash only.